### ISSUE TITLE
Idea: Use openvoronoi as alternative toolpath in F-Engrave

### ISSUE THREAD

**Issue by Blokkendoos** [aewallin/openvoronoi#34]
Hi Anders,
In the works of refactoring F-Engrave I got the idea to add openvoronoi as an alternative toolpath strategy. After having consulted Scorch, the author, I renamed F-Engrave to OOF-Engrave.

Both openvoronoi as f-engrave are under GPLv3 so I expect using the first in the latter is no problem. My question is whether this assumption is true.

**Comment by aewallin** [comment]
Yes you are correct there should be no issues at all if both programs/libraries are GPL licensed.
For openvoronoi it should be possible to build the library and just link against it from the project that is using the library. I don't remember but there might be an openvoronoi-dev package already for this (it installs the header-files also).

do you have a link to pictures or videos of the toolpaths that OOF-Engrave produces?

**Comment by Blokkendoos** [comment]
Thanks for your swift response.
OOF-Engrave is a Python application. My idea is to use OV as shown in the python examples and, for V-carving, the medial axis examples in particular.
OOF-Engrave 2D toolpath example: https://youtu.be/Xz_Q1KmSoDI
![schermafbeelding 2018-03-28 om 19 03 25](https://user-images.githubusercontent.com/633633/38051741-791cdc24-32cf-11e8-90a1-5617ff34dfbf.png)

**Comment by Blokkendoos** [comment]
NC
